packages/common/src/intl/locales/en/common.json

```diff
@@ -1,4 +1,5 @@
 {
+  "app": "mSupply",
```

Comment:
Thought this seemed appropriate for the "root" app name, though can change to `app.name` if that's preferable.

Reply:
I like `app.name` but it would be confusing next to the other entries. what you have is probably best